### PR TITLE
Support rejson module commands

### PR DESCRIPTION
**Pull request by kishorekasi**
- [x] Add support to parse rejson module commands in dynomite
- [x] Add command tests.

### CODE REVIEW

File: src/dyn_message.h [hunk 1]
```diff
@@ -211,25 +212,42 @@ typedef enum msg_parse_result {
...
-                      ACTION(RSP_REDIS_ERROR_MISCONF)                          \
-                          ACTION(RSP_REDIS_ERROR_NOSCRIPT)                     \
-                              ACTION(RSP_REDIS_ERROR_READONLY)                 \
-                                  ACTION(RSP_REDIS_ERROR_WRONGTYPE) ACTION(    \
-                                      RSP_REDIS_ERROR_EXECABORT)               \
-                                      ACTION(RSP_REDIS_ERROR_MASTERDOWN)       \
-                                          ACTION(RSP_REDIS_ERROR_NOREPLICAS)   \
-                                              ACTION(SENTINEL)
+  ACTION(REQ_REDIS_GEORADIUSBYMEMBER)                                          \
+  ACTION(REQ_REDIS_UNLINK)                                                     \
```
Comment (ipapapa) [inline src/dyn_message.h:216]: We are not really implementing all these commands below. Is there a need to create an `ACTION`?

**Comment by smukil** [comment]
Will have a look by EOD.

**Comment by smukil** [comment]
Code LGTM. It will be good to add some tests.

**Comment by kishorekasi** [comment]
I have added dyno tests for json recipe

**Pull request merged**